### DISCUSSION TITLE
I use draw_geometries_with_animation_callback to display a animation, but i can not save it to a video.how to do it

### DISCUSSION THREAD

**Discussion by yier2333** [isl-org/Open3D#3822] [Q&A] [answered]
I use the code in the example and successful display of animation, then i want to save the animation to a  video, i do not know how to do it 

`def custom_draw_geometry_with_rotation(pcd, fov_step=-90):
    def rotate_view(vis):
        ctr = vis.get_view_control()
        ctr.rotate(3, 0)
        return False
    o3d.visualization.draw_geometries_with_animation_callback([pcd],
                                                              rotate_view, width=1080, height=1080)`

**Comment by ssheorey** [comment, marked as answer]
You can try screen capture / screen recorder software. macOS, Linux (Gnome) and Windows 10 all have this built in.
